dts/arm/nuvoton/npcx.dtsi

```diff
@@ -101,6 +101,20 @@
 			label = "SCFG";
 		};
 
+		mdc: mdc@4000c000 {
+			compatible = "syscon";
+			reg = <0x4000c000 0xa>;
+			reg-io-width = <1>;
+			label = "MDC";
```

Comment:
Are the labels needed?

Reply:
I'm not sure it's _needed_ but it does follow the pattern in the rest of the file so I think we shouldn't deviate.

Reply:
We are actually trying to move away from labels so remove it if you do not plan to use them.

Reply:
Done